<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoteApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNoteButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Notiz hinzufügen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayNotesButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Notizen anzeigen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editNoteButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Notiz bearbeiten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteNoteButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Notiz löschen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNewNote"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNewNote">
   <property name="text">
    <string>Neue Notiz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
